backend/app/services/flow.py

```
@@ -0,0 +1,1011 @@
+            try:
+                from zoneinfo import ZoneInfo
+
+                from croniter import croniter
+
+                cron_expr = trigger_config.get("expression", "0 9 * * *")
+                timezone_str = trigger_config.get("timezone", "UTC")
+
+                # Get the user's timezone
+                try:
+                    user_tz = ZoneInfo(timezone_str)
+                except Exception:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)